draft-ietf-rtcweb-jsep.xml

```diff
@@ -454,6 +454,27 @@ setRemote(ANSWER) |                                   |
                 application will not need to read or change them.</t>
 
             </section>
+            <section title="Session Description Versions"
+                     anchor="sec.session-description-vers">
+              <t>
+                The SessionDescription contains a "session-version"
+                field which is required by <xref target="RFC4566"/> to increase with
```

Comment:
4566 requires it to increase
   <sess-version> is a version number for this session description.  Its
      usage is up to the creating tool, so long as <sess-version> is
      increased when a modification is made to the session data.  Again,
      it is RECOMMENDED that an NTP format timestamp is used.